packages/postcss-minify-font-values/src/lib/minify-font.js

```diff
@@ -56,9 +56,31 @@ module.exports = function (nodes, opts) {
     }
   }
 
-  familyStart += 2;
+  let needAdditionalSpace = false;
 
-  family = minifyFamily(nodes.slice(familyStart), opts);
+  // handle case .8em"Times New Roman"
+  if (nodes[familyStart + 1].type === 'space') {
+    familyStart += 2;
+  } else {
+    // @ts-ignore
```

Comment:
I don't think we use `ts-ignore` anywhere else. Could you find a way to solve this through an additional check or a TypeScript cast?

Reply:
The tests do not pass either, there's likely too many assumptions about the shape of the nodes here.